src/platform/tigerlake/include/platform/lib/clk.h

```diff
@@ -24,11 +24,7 @@
 
 #define CPU_LOWEST_FREQ_IDX CPU_WOVCRO_FREQ_IDX
 
-#if CONFIG_CAVS_LPRO_ONLY
-#define CPU_DEFAULT_IDX CPU_LPRO_FREQ_IDX
-#else
 #define CPU_DEFAULT_IDX CPU_HPRO_FREQ_IDX
-#endif
```

Comment:
we should remove this too then: https://github.com/thesofproject/sof/blob/61e1fb1a4bb04545e4139c86d3e6df0b20dff64b/src/platform/Kconfig#L357-L364

Reply:
sure, thanks